pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -15,13 +15,6 @@ import (
 )
 
 var genericPreflightChecks = [...]Check{
-	{
-		configKeySuffix:  "check-oc-cached",
-		checkDescription: "Checking if oc binary is cached",
-		check:            checkOcBinaryCached,
-		fixDescription:   "Caching oc binary",
-		fix:              fixOcBinaryCached,
-	},
```

Comment:
Once you do https://github.com/code-ready/crc/pull/1630, we could have a preflight running only at `crc start` time ensuring an `oc` binary is available.

Reply:
yes!